stdlib/public/core/FloatingPoint.swift

```diff
@@ -328,15 +328,24 @@ public protocol FloatingPoint: SignedNumeric, Strideable, Hashable
   /// `infinity` is greater than this value.
   static var greatestFiniteMagnitude: Self { get }
 
-  /// The mathematical constant pi.
+  /// The [mathematical constant π][wiki], approximately equal to 3.14159.
   ///
-  /// This value should be rounded toward zero to keep user computations with
-  /// angles from inadvertently ending up in the wrong quadrant. A type that
-  /// conforms to the `FloatingPoint` protocol provides the value for `pi` at
-  /// its best possible precision.
+  /// This constant is most often defined as the numerical value of the ratio
+  /// of a circle's circumference to its diameter, which is the same for all
+  /// circles.
+  ///
+  /// A common application of this constant in geometry is to measure a half
+  /// revolution in radians. Used like this, it is equivalent to 180 degrees.
+  ///
+  /// This value is rounded toward zero to keep user computations with angles
+  /// from inadvertently ending up in the wrong quadrant. A type that conforms
+  /// to the `FloatingPoint` protocol provides the value for `pi` at its best
+  /// possible precision.
   ///
   ///     print(Double.pi)
   ///     // Prints "3.14159265358979"
+  ///
+  /// [wiki]: https://en.wikipedia.org/wiki/Pi
```

Comment:
nitpick: instead of `[wiki]`, maybe `["π" on Wikipedia]` or `[Wikipedia article on π]` is more clear as the hover-text to the reader.

Reply:
Is this really used for the reader? Looking at the rendered HTML I don't see the link tag text being included anywhere for the existing links.

For example in the module description there's a `[IEEE 754 specification][spec]` link that renders as `<a href="http://ieeexplore.ieee.org/servlet/opac?punumber=4610933">IEEE 754 specification</a>` - the `spec` tag is discarded and only the URL is used. But maybe there's a special version that gets served to devices with screen readers?

Reply:
Yeah, I think the existing tag is fine unless there's a specific example where it affects hover text.

Reply:
My mistake. I just checked [the spec](https://github.github.com/gfm/#link-label), and indeed the link label doesn't become the hover text. For some reason I always thought it did.